opsmngr/automation_config.go

```diff
@@ -96,6 +96,16 @@ type AutomationConfig struct {
 	Version            int                       `json:"version,omitempty"`
 }
 
+// IndexConfiguration represents a new index requests for a given database and collection.
+type IndexConfigs struct {
+	DBName         string                  `json:"dbName"`              // Database that is indexed
+	CollectionName string                  `json:"collectionName"`      // Collection that is indexed
+	RSName         string                  `json:"rsName"`              // The replica set that the index is built on
+	Key            [][]string              `json:"key"`                 // Keys array of keys to index and their type, sorting of keys is important for an index
```

Comment:
[q] do the keys not have options? can you give me an example of the values here (just trying to picture this) would it be something like: `[ ["name", "text"], ["ssn", "1"] ]` ?

Reply:
yes, it is going to be something like that. 
This command `./bin/mongocli om clusters indexes create --collectionName test --db test --rsName myReplicaSet --key name:1 --key  surname:1  --projectId 5e9d600f338fa44c13bc6cbf --profile OPS`

creates a compound index where the keys are stored in this way [ ["name", "1"], ["surname", "1"] ]